src/components/listBox/ListBox.tsx

```diff
@@ -69,7 +70,7 @@ export class ListBox extends React.Component<IListBoxProps, {}> {
             .map((item: IItemBoxProps) => <ItemBox
                 key={item.value}
                 {...item}
-                onOptionClick={(option: IItemBoxProps) => this.onSelectItem(option)}
+                onOptionClick={(option: IItemBoxProps) => this.onSelectItem(option, item)}

```

Comment:
Why are you sending both the option and the item?
IMO you should use: `onOptionClick={() => this.onSelectItem(item)}`

Reply:
It would be a breaking change, he needs option in the `onSelectItem` method

Reply:
its not the same object item and option. I think its can break something if we change that

Reply:
It's not a breaking change since we're sending an element with the same type